### PR TITLE
Localization & French Translation

### PR DESCRIPTION
**Pull request by zicklag**
Depends on #58.

Since it didn't take me long and it's better to do earlier so we don't have to re-do all our strings later, I setup localization!

Currently there's a French and English translation and it's super easy to add new translations. It will auto-detect the language from the users system, even in the browser, which saves us from making a language selection menu for now. :)

### CODE REVIEW

**Comment by zicklag** [comment]
This should be ready now. Once https://github.com/kgv/bevy_fluent/pull/21 is merged, we can update the `bevy_fluent` git dependency and we'll get hot reload for the translations.

**Comment by zicklag** [comment]
Found that there is a translation selection issue on WASM actually, got to fix that first.

**Comment by zicklag** [comment]
OK, fixed it. Should be good to go now.

File: src/assets.rs [hunk 1]
```diff
@@ -23,13 +23,6 @@ pub fn register(app: &mut bevy::prelude::App) {
         .add_asset_loader(EguiFontLoader);
 }
 
-// An error that could ocurr during asset processing
-#[derive(thiserror::Error, Debug)]
-pub enum AssetLoaderError {
```
Comment (odecay) [inline src/assets.rs:28]: Just checking to make sure this is no longer necessary or useful..

Comment (zicklag) [reply src/assets.rs:28]: Yeah, it wasn't actually used. It was used in the code I had that I copied the old loader from, but it didn't end up being incorporated.

**Pull request merged**